src/test/java/org/apache/commons/configuration2/TestAbstractConfiguration.java

```diff
@@ -78,6 +78,49 @@ public void testAddPropertyDirect() {
         assertEquals(expected, list);
     }
 
+    @Test
+    void testContainsValue(){
+        Configuration config = getConfiguration();
+        assertTrue(config.containsValue("value1"), "should return true for class " + this.getClass());
+        assertFalse(config.containsValue("value99999"), "should return false for class " + this.getClass());
+    }
+
+    @Test
+    void testContains(){
+        AbstractConfiguration config = getConfiguration();
+
```

Comment:
No need to blank lines. See my previous comment.

Reply:
Corrected.